build/repo.targets

```diff
@@ -876,30 +852,21 @@
       Command="docker build --build-arg USER_ID=%24(id -u) -t docker-image-$(InstallerPlatform) $(InstallerPlatform)"
       WorkingDirectory="$(DockerDir)" />
 
-    <!-- Parse CLI version -->
-    <ReadLinesFromFile File="$(BuildDirectory)cli.version">
-      <Output TaskParameter="Lines" PropertyName="CLIVersionNumber" />
-    </ReadLinesFromFile>
+    <!-- Delete korebuild-lock.txt to trigger re-installation of dotnet -->
+    <Delete Files="$(RepositoryRoot)korebuild-lock.txt" />
```

Comment:
why? This is potential problematic. 😬 The lockfile should work without changes, even within the docker container.